app/.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```

Comment:
IntelliSense is part of the JetBrains IDEs, was this done in Intellij?

Reply:
Created this project using Android Studio and as you already know android studio uses jetbrain

Reply:
Did you install Flutter using command line, or is it a plugin for android studio?

Reply:
I installed it using android studio